aspnetcore/mvc/controllers/routing.md

```diff
@@ -70,667 +65,788 @@ public class HomeController : Controller
 }
 ```
 
-Using this controller definition and route template, the `HomeController.Index` action would be executed for any of the following URL paths:
+Using the preceding controller definition and route template, the `HomeController.Index` action is run for the following URL paths:
 
 * `/Home/Index/17`
-
 * `/Home/Index`
-
 * `/Home`
-
 * `/`
 
-The convenience method `UseMvcWithDefaultRoute`:
+The URL path `/` uses the route template default `Home` controllers and `Index` action. The URL path `/Home` uses the route template default `Index` action.
+
+The convenience method <xref:Microsoft.AspNetCore.Builder.ControllerEndpointRouteBuilderExtensions.MapDefaultControllerRoute*>:
 
 ```csharp
 endpoints.MapDefaultControllerRoute();
 ```
 
-Can be used to replace:
+Replaces:
 
 ```csharp
 endpoints.MapControllerRoute("default", "{controller=Home}/{action=Index}/{id?}");
 ```
 
-Routing is configured using the `UseRouting()` and `UseEndpoints()` middleware. To use controllers:
-* Call `MapControllers()` inside `UseEndpoints()` to map attribute-routed controllers.
-* Call `MapControllerRoute()` (or similar) to map conventionally-routed controllers.
+Routing is configured using the <xref:Microsoft.AspNetCore.Builder.EndpointRoutingApplicationBuilderExtensions.UseRouting*> and <xref:Microsoft.AspNetCore.Builder.EndpointRoutingApplicationBuilderExtensions.UseEndpoints*> middleware. To use controllers:
 
-Using `MapControllerRoute()` (or similar) will also map attribute-routed controllers.
+* Call <xref:Microsoft.AspNetCore.Builder.ControllerEndpointRouteBuilderExtensions.MapControllers*> inside `UseEndpoints` to map [attribute routed](#ar) controllers.
+* Call <xref:Microsoft.AspNetCore.Builder.ControllerEndpointRouteBuilderExtensions.MapControllerRoute*> or <xref:Microsoft.AspNetCore.Builder.ControllerEndpointRouteBuilderExtensions.MapAreaControllerRoute*>, to map [conventionally routed](#cr) controllers.
 
 <a name="routing-conventional-ref-label"></a>
+<a name="cr"></a>
 
-## Conventional routing
+## Conventional endpoint routing
```

Comment:
This should be reverted to `Conventional routing` (here and everywhere else it appears. `Conventional endpoint routing` is not a term I want to use.